components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -24,15 +25,22 @@ function TeamUsage() {
     const [billedUsage, setBilledUsage] = useState<BillableSession[]>([]);
     const [currentPage, setCurrentPage] = useState(1);
     const [resultsPerPage] = useState(10);
+    const [errorMessage, setErrorMessage] = useState("");
 
     useEffect(() => {
         if (!team) {
             return;
         }
         (async () => {
             const attributionId = AttributionId.render({ kind: "team", teamId: team.id });
-            const billedUsageResult = await getGitpodService().server.listBilledUsage(attributionId);
-            setBilledUsage(billedUsageResult);
+            try {
+                const billedUsageResult = await getGitpodService().server.listBilledUsage(attributionId);
+                setBilledUsage(billedUsageResult);
+            } catch (error) {
+                if (error.code === ErrorCodes.PERMISSION_DENIED) {
+                    setErrorMessage("Please request access to your administrator.");
```

Comment:
```suggestion
                    setErrorMessage("Please request access from your team Owner.");
```
Since that's the role we're using